server/liveblog/syndication/blogs.py

```diff
@@ -107,16 +129,26 @@ def _delete_blogs_syndicate(blog_id, consumer_blog_id):
         return api_response({}, 204)
 
 
-@blogs_blueprint.route('/api/syndication/blogs/<string:blog_id>/syndicate', methods=['POST', 'DELETE'])
+@blogs_blueprint.route('/api/syndication/blogs/<string:blog_id>/syndicate', methods=['POST', 'PATCH', 'DELETE'])
 def blogs_syndicate(blog_id):
-    consumer_blog_id = request.get_json().get('consumer_blog_id')
+    data = request.get_json()
+    start_date = data.get('start_date')
+    if start_date:
+        try:
+            start_date = str_to_date(start_date)
```

Comment:
Might be nice to document somewhere, even if just in a comment, what is the expected format